### ISSUE TITLE
something went wrong

### ISSUE THREAD

**Issue by pheonix3082** [TeamNewPipe/NewPipe#8796]
### Checklist

- [X] I am able to reproduce the bug with the [latest version](https://github.com/TeamNewPipe/NewPipe/releases/latest).
- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.
- [X] This issue contains only one bug.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Affected version

v0.23.2

### Steps to reproduce the bug

>Open newpipe
>tapped first video in tending
>bug shows up

### Expected behavior

Video to load and play

### Actual behavior

Bug (couldn't parse video)

## Exception
* __User Action:__ requested stream
* __Request:__ https://www.youtube.com/watch?v=SFjGE4QXYqk
* __Content Country:__ GB
* __Content Language:__ en-GB
* __App Language:__ en_GB
* __Service:__ YouTube
* __Version:__ 0.23.1
* __OS:__ Linux Android 12 - 32
<details><summary><b>Crash log </b></summary><p>

```
org.schabi.newpipe.extractor.stream.StreamInfo$StreamExtractException: Could not get any stream. See error variable to get further details.
	at org.schabi.newpipe.extractor.stream.StreamInfo.extractStreams(StreamInfo.java:188)
	at org.schabi.newpipe.extractor.stream.StreamInfo.getInfo(StreamInfo.java:82)
	at org.schabi.newpipe.extractor.stream.StreamInfo.getInfo(StreamInfo.java:73)
	at org.schabi.newpipe.util.ExtractorHelper.lambda$getStreamInfo$3(ExtractorHelper.java:118)
	at org.schabi.newpipe.util.ExtractorHelper.$r8$lambda$YTHJjScxCJNO1LTCqs3IKy35iyY(Unknown Source:0)
	at org.schabi.newpipe.util.ExtractorHelper$$ExternalSyntheticLambda6.call(Unknown Source:4)
	at io.reactivex.rxjava3.internal.operators.single.SingleFromCallable.subscribeActual(SingleFromCallable.java:43)
	at io.reactivex.rxjava3.core.Single.subscribe(Single.java:4813)
	at io.reactivex.rxjava3.internal.operators.single.SingleDoOnSuccess.subscribeActual(SingleDoOnSuccess.java:35)
	at io.reactivex.rxjava3.core.Single.subscribe(Single.java:4813)
	at io.reactivex.rxjava3.internal.operators.maybe.MaybeFromSingle.subscribeActual(MaybeFromSingle.java:41)
	at io.reactivex.rxjava3.core.Maybe.subscribe(Maybe.java:5330)
	at io.reactivex.rxjava3.internal.operators.maybe.MaybeConcatArray$ConcatMaybeObserver.drain(MaybeConcatArray.java:153)
	at io.reactivex.rxjava3.internal.operators.maybe.MaybeConcatArray$ConcatMaybeObserver.request(MaybeConcatArray.java:78)
	at io.reactivex.rxjava3.internal.operators.flowable.FlowableElementAtMaybe$ElementAtSubscriber.onSubscribe(FlowableElementAtMaybe.java:66)
	at io.reactivex.rxjava3.internal.operators.maybe.MaybeConcatArray.subscribeActual(MaybeConcatArray.java:42)
	at io.reactivex.rxjava3.core.Flowable.subscribe(Flowable.java:15868)
	at io.reactivex.rxjava3.internal.operators.flowable.FlowableElementAtMaybe.subscribeActual(FlowableElementAtMaybe.java:36)
	at io.reactivex.rxjava3.core.Maybe.subscribe(Maybe.java:5330)
	at io.reactivex.rxjava3.internal.operators.maybe.MaybeToSingle.subscribeActual(MaybeToSingle.java:46)
	at io.reactivex.rxjava3.core.Single.subscribe(Single.java:4813)
	at io.reactivex.rxjava3.internal.operators.single.SingleSubscribeOn$SubscribeOnObserver.run(SingleSubscribeOn.java:89)
	at io.reactivex.rxjava3.core.Scheduler$DisposeTask.run(Scheduler.java:644)
	at io.reactivex.rxjava3.internal.schedulers.ScheduledRunnable.run(ScheduledRunnable.java:65)
	at io.reactivex.rxjava3.internal.schedulers.ScheduledRunnable.call(ScheduledRunnable.java:56)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:301)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
	at java.lang.Thread.run(Thread.java:920)

```
</details>
<hr>

### Affected Android/Custom ROM version

CalyxOS

### Affected device model

google pixel 5

**Comment by baldarn** [comment]
Same error here

`
{"user_action":"requested stream","request":"https://www.youtube.com/watch?v=BDiNNU3ZUKQ","content_language":"it-IT","content_country":"IT","app_language":"it_IT","service":"YouTube","package":"org.schabi.newpipe","version":"0.23.1","os":"Linux Xiaomi/apollo_eea/apollo:12/SKQ1.211006.001/V13.0.7.0.SJDEUXM:user/release-keys 12 - 31","time":"2022-08-15 11:14","exceptions":["org.schabi.newpipe.extractor.stream.StreamInfo$StreamExtractException: Could not get any stream. See error variable to get further details.\n\tat org.schabi.newpipe.extractor.stream.StreamInfo.extractStreams(StreamInfo.java:188)\n\tat org.schabi.newpipe.extractor.stream.StreamInfo.getInfo(StreamInfo.java:82)\n\tat org.schabi.newpipe.extractor.stream.StreamInfo.getInfo(StreamInfo.java:73)\n\tat org.schabi.newpipe.util.ExtractorHelper.lambda$getStreamInfo$3(ExtractorHelper.java:118)\n\tat org.schabi.newpipe.util.ExtractorHelper.$r8$lambda$YTHJjScxCJNO1LTCqs3IKy35iyY(Unknown Source:0)\n\tat org.schabi.newpipe.util.ExtractorHelper$$ExternalSyntheticLambda6.call(Unknown Source:4)\n\tat io.reactivex.rxjava3.internal.operators.single.SingleFromCallable.subscribeActual(SingleFromCallable.java:43)\n\tat io.reactivex.rxjava3.core.Single.subscribe(Single.java:4813)\n\tat io.reactivex.rxjava3.internal.operators.single.SingleDoOnSuccess.subscribeActual(SingleDoOnSuccess.java:35)\n\tat io.reactivex.rxjava3.core.Single.subscribe(Single.java:4813)\n\tat io.reactivex.rxjava3.internal.operators.maybe.MaybeFromSingle.subscribeActual(MaybeFromSingle.java:41)\n\tat io.reactivex.rxjava3.core.Maybe.subscribe(Maybe.java:5330)\n\tat io.reactivex.rxjava3.internal.operators.maybe.MaybeConcatArray$ConcatMaybeObserver.drain(MaybeConcatArray.java:153)\n\tat io.reactivex.rxjava3.internal.operators.maybe.MaybeConcatArray$ConcatMaybeObserver.request(MaybeConcatArray.java:78)\n\tat io.reactivex.rxjava3.internal.operators.flowable.FlowableElementAtMaybe$ElementAtSubscriber.onSubscribe(FlowableElementAtMaybe.java:66)\n\tat io.reactivex.rxjava3.internal.operators.maybe.MaybeConcatArray.subscribeActual(MaybeConcatArray.java:42)\n\tat io.reactivex.rxjava3.core.Flowable.subscribe(Flowable.java:15868)\n\tat io.reactivex.rxjava3.internal.operators.flowable.FlowableElementAtMaybe.subscribeActual(FlowableElementAtMaybe.java:36)\n\tat io.reactivex.rxjava3.core.Maybe.subscribe(Maybe.java:5330)\n\tat io.reactivex.rxjava3.internal.operators.maybe.MaybeToSingle.subscribeActual(MaybeToSingle.java:46)\n\tat io.reactivex.rxjava3.core.Single.subscribe(Single.java:4813)\n\tat io.reactivex.rxjava3.internal.operators.single.SingleSubscribeOn$SubscribeOnObserver.run(SingleSubscribeOn.java:89)\n\tat io.reactivex.rxjava3.core.Scheduler$DisposeTask.run(Scheduler.java:644)\n\tat io.reactivex.rxjava3.internal.schedulers.ScheduledRunnable.run(ScheduledRunnable.java:65)\n\tat io.reactivex.rxjava3.internal.schedulers.ScheduledRunnable.call(ScheduledRunnable.java:56)\n\tat java.util.concurrent.FutureTask.run(FutureTask.java:266)\n\tat java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:301)\n\tat java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)\n\tat java.lang.Thread.run(Thread.java:920)\n"],"user_comment":""}
`

**Comment by pheonix3082** [comment]
Ah OK. Sorry for double post

On Monday, August 15, 2022, lorenzo farnararo ***@***.***>
wrote:
> Same error here
>
> {"user_action":"requested stream","request":"
https://www.youtube.com/watch?v=BDiNNU3ZUKQ","content_language":"it-IT","content_country":"IT","app_language":"it_IT","service":"YouTube","package":"org.schabi.newpipe","version":"0.23.1","os":"Linux
Xiaomi/apollo_eea/apollo:12/SKQ1.211006.001/V13.0.7.0.SJDEUXM:user/release-keys
12 - 31","time":"2022-08-15
11:14","exceptions":["org.schabi.newpipe.extractor.stream.StreamInfo$StreamExtractException:
Could not get any stream. See error variable to get further details.\n\tat
org.schabi.newpipe.extractor.stream.StreamInfo.extractStreams(StreamInfo.java:188)\n\tat
org.schabi.newpipe.extractor.stream.StreamInfo.getInfo(StreamInfo.java:82)\n\tat
org.schabi.newpipe.extractor.stream.StreamInfo.getInfo(StreamInfo.java:73)\n\tat
org.schabi.newpipe.util.ExtractorHelper.lambda$getStreamInfo$3(ExtractorHelper.java:118)\n\tat
org.schabi.newpipe.util.ExtractorHelper.$r8$lambda$YTHJjScxCJNO1LTCqs3IKy35iyY(Unknown
Source:0)\n\tat
org.schabi.newpipe.util.ExtractorHelper$$ExternalSyntheticLambda6.call(Unknown
Source:4)\n\tat
io.reactivex.rxjava3.internal.operators.single.SingleFromCallable.subscribeActual(SingleFromCallable.java:43)\n\tat
io.reactivex.rxjava3.core.Single.subscribe(Single.java:4813)\n\tat
io.reactivex.rxjava3.internal.operators.single.SingleDoOnSuccess.subscribeActual(SingleDoOnSuccess.java:35)\n\tat
io.reactivex.rxjava3.core.Single.subscribe(Single.java:4813)\n\tat
io.reactivex.rxjava3.internal.operators.maybe.MaybeFromSingle.subscribeActual(MaybeFromSingle.java:41)\n\tat
io.reactivex.rxjava3.core.Maybe.subscribe(Maybe.java:5330)\n\tat
io.reactivex.rxjava3.internal.operators.maybe.MaybeConcatArray$ConcatMaybeObserver.drain(MaybeConcatArray.java:153)\n\tat
io.reactivex.rxjava3.internal.operators.maybe.MaybeConcatArray$ConcatMaybeObserver.request(MaybeConcatArray.java:78)\n\tat
io.reactivex.rxjava3.internal.operators.flowable.FlowableElementAtMaybe$ElementAtSubscriber.onSubscribe(FlowableElementAtMaybe.java:66)\n\tat
io.reactivex.rxjava3.internal.operators.maybe.MaybeConcatArray.subscribeActual(MaybeConcatArray.java:42)\n\tat
io.reactivex.rxjava3.core.Flowable.subscribe(Flowable.java:15868)\n\tat
io.reactivex.rxjava3.internal.operators.flowable.FlowableElementAtMaybe.subscribeActual(FlowableElementAtMaybe.java:36)\n\tat
io.reactivex.rxjava3.core.Maybe.subscribe(Maybe.java:5330)\n\tat
io.reactivex.rxjava3.internal.operators.maybe.MaybeToSingle.subscribeActual(MaybeToSingle.java:46)\n\tat
io.reactivex.rxjava3.core.Single.subscribe(Single.java:4813)\n\tat
io.reactivex.rxjava3.internal.operators.single.SingleSubscribeOn$SubscribeOnObserver.run(SingleSubscribeOn.java:89)\n\tat
io.reactivex.rxjava3.core.Scheduler$DisposeTask.run(Scheduler.java:644)\n\tat
io.reactivex.rxjava3.internal.schedulers.ScheduledRunnable.run(ScheduledRunnable.java:65)\n\tat
io.reactivex.rxjava3.internal.schedulers.ScheduledRunnable.call(ScheduledRunnable.java:56)\n\tat
java.util.concurrent.FutureTask.run(FutureTask.java:266)\n\tat
java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:301)\n\tat
java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)\n\tat
java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)\n\tat
java.lang.Thread.run(Thread.java:920)\n"],"user_comment":""}
>
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you authored the thread.<
https://ci5.googleusercontent.com/proxy/lqvYM5eOeOrPjLxr7IYjzjGHQbChQDRyXDQCZWM9FZ2j6mtzWj-q_Cw-bTWAEpb9yss65ad4XyznZ70FHP0BrtagrvUzYUzr4R74w4kOSq-qn0yF1oq3-K23H0NcwFHd9TlM3JxLtfqbMquT-qDKiHrMElGtJqmFSTX61BK_fJSllPnpCnMDucELBfFepAuNiNFbEwlyqnqemskrb1nCLDiGwevKyxVswIoMPTUWCw=s0-d-e1-ft#https://github.com/notifications/beacon/AS5OSIKVLHBNEDIGYNCLHALVZIDDPA5CNFSM56RSF2KKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOJBUFLBY.gif>Message
ID: ***@***.***>

**Comment by mhmdanas** [comment]
@pheonix3082 you stated that your NewPipe version is 0.23.2 in the issue template, but the error report states that you are actually using 0.23.1, and this specific error was fixed in 0.23.2. Please upgrade.

Closing as duplicate of #8760.

**Comment by pheonix3082** [comment]
Apologies, I stand corrected.

On Monday, August 15, 2022, Mohammed Anas ***@***.***> wrote:
> @pheonix3082 you stated that your NewPipe version is 0.23.2 in the issue
template, but the error report states that you are actually using 0.23.1,
and this specific error was fixed in 0.23.2. Please upgrade.
>
> Closing as duplicate of #8760.
>
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you were mentioned.<
https://ci4.googleusercontent.com/proxy/1SedPSgTKL6ie8_z1nqkwgCaPsUC3EubkffvSUVj_JzEHQbYrs1MyNfbxTlzK0jxFmMciyyEPRAmKoT9OFJxMRDSs0MELlUreu3wXQPm52w9Id8O0ENbuhNO1VGdhFAkO0TGwlV3PFoYE8RO8iBkisTaTfNriA6V-QKR59RSGENOqbCqmh9V5pDYRA7wLCNyh9lxNU1BvVwAvIH_3nSlpnJQZ85h5jsaBbCQj6c_rg=s0-d-e1-ft#https://github.com/notifications/beacon/AS5OSIJPWXVT3FOTGOI52DDVZIG3TA5CNFSM56RSF2KKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOJBUNVTA.gif>Message
ID: ***@***.***>
